client/src/container/Leads.jsx

```diff
@@ -33,61 +36,13 @@ class Leads extends React.Component {
               settings={{
                 licenseKey: '7fb69-d3720-89c63-24040-8e45b',
                 data: this.props.leads,
-                dataSchema: {
-                  id: null,
-                  ownerId: null,
-                  description: null,
-                  firstName: null,
-                  lastName: null,
-                  suffix: null,
-                  title: null,
-                  value: null,
-                  email: null,
-                  phoneNumber: null,
-                  createdAt: null
-                },
-                colHeaders: [
-                  'id',
-                  'ownerId',
-                  'description',
-                  'firstName',
-                  'lastName',
-                  'suffix',
-                  'title',
-                  'value',
-                  'email',
-                  'phoneNumber',
-                  'createdAt'
-                ],
-                columns: [
-                  { data: 'id' },
-                  { data: 'ownerId' },
-                  { data: 'description' },
-                  {
-                    data: 'firstName'
-                  },
-                  { data: 'lastName' },
-                  { data: 'suffix' },
-                  { data: 'title' },
-                  {
-                    data: 'value',
-                    type: 'numeric',
-                    format: '$0,0.00'
-                  },
-                  { data: 'email' },
-                  { data: 'phoneNumber' },
-                  {
-                    data: 'createdAt',
-                    type: 'date',
-                    dateFormat: 'MM/DD/YYYY',
-                    correctFormat: true
-                    // readOnly: true
-                  }
-                ],
+                colHeaders: this.props.leadsColumsHeader,
```

Comment:
Did you intentionally name the property leadsColumsHeader (without an 'n' at end of column)?   All other properties and variable names you use (outside of reducer and mapStateToProps) that contain the word column spell it correctly, with an N at the end.